README.md

```diff
@@ -90,6 +97,14 @@ You do not have sufficient access to the device. You probably want to add the a
 
 <!-- markdownlint-enable MD013 -->
 
+## Interference from CUPS
```

Comment:
I thought you were still having issues after disabling CUPS: https://github.com/labelle-org/labelle/issues/68#issuecomment-2273356623

Also any ideas why you're the first to report this? Could it be specific to the 280?

Reply:
Very occasionally.  Before disabling CUPS, labelle was nearly unusable.  Now I occasionally see that, let's call it maybe 2% of the time instead of being easily reproducible almost every time.  I believe it to be a different issue.

Reply:
Similarly, could you please explicitly mention the overflow error in relation to this workaround?

Reply:
Done in 2dd0b48

Reply:
Thanks!